python/lsst/ctrl/orca/CondorWorkflowMonitor.py

```diff
@@ -173,7 +173,7 @@ def run(self):
                         print("logger handled... and... done!")
                         return
 
-                if (event is not None) or (logEvent is not None):
+                if not event or not logEvent:
```

Comment:
I thought our Developer Guide does want us to compare with `None` explicitly? (https://developer.lsst.io/coding/python_style_guide.html#is-and-is-not-should-be-used-when-comparing-to-none)